internal/controller/resourceset_controller.go

```diff
@@ -456,23 +456,21 @@ func (r *ResourceSetReconciler) apply(ctx context.Context,
 			"kustomize.toolkit.fluxcd.io/name",
 			"kustomize.toolkit.fluxcd.io/namespace",
 		},
-		// Take ownership of the Flux resources if they
-		// were previously managed by other tools.
+		// Undo changes made by kubectl.
 		FieldManagers: []ssa.FieldManager{
 			{
-				Name:          "flux",
-				OperationType: metav1.ManagedFieldsOperationApply,
-			},
-			{
-				Name:          "kustomize-controller",
+				// to undo changes made with 'kubectl apply --server-side --force-conflicts'
+				Name:          "kubectl",
 				OperationType: metav1.ManagedFieldsOperationApply,
 			},
 			{
-				Name:          "helm",
```

Comment:
We won't under changes from helm and kustomize-controller anymore?

Reply:
This break both controllers as their finalizers are removed by this.